docs/docs/deep-dive/override-os-files.md

```diff
@@ -15,7 +15,7 @@ Tracee will show you collected information about the running Linux OS with the
 `--log debug` argument:
 
 ```
-$ sudo ./dist/tracee-ebpf --log debug --filter uid=1000 --filter pid=new --filter event=execve
```

Comment:
we said this file would move to installation

Reply:
done